__tests__/hooks/useGroupSetting.test.js

```diff
@@ -18,6 +17,12 @@ jest.mock('react-native-simple-toast', () => ({
   show: jest.fn()
 }));
 
+jest.mock('react-native-compressor', () => ({
+  Image: {
+    compressor: jest.fn()
+  }
+}));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new mock for the 'react-native-compressor' library has been added. This seems to be a good addition if you're planning to test functionality related to image compression. Just make sure that the mocked function behaves as expected in your tests.
```diff
+ jest.mock('react-native-compressor', () => ({
+   Image: {
+     compressor: jest.fn()
+   }
+ }));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->